matlab_functions/structure/StructureDesignerRibWing.m

```diff
@@ -841,12 +841,12 @@
     %% recalculate mass distribution
     damping=0.4;
     % wing mass distr
-    wing_m_distr =damping*wing_m_distr + (1-damping)*correction_factor*(1.2*ones(size(y,2),1)*delta*(...
+    wing_m_distr =damping*wing_m_distr + (1-damping)*correction_factor*1.2*ones(size(y,2),1)*delta*(...
```

Comment:
I did not implement this, but I had a look at this. I think both the version that you posted and the old version are not fully correct. The correct one should be:

```matlab
wing_m_distr =damping*wing_m_distr + (1-damping)*correction_factor*(1.2*ones(size(y,2),1)*delta*(...
        rho_mat(6)*t_skin_wing*s0*c... %wing skin
        +2*h_prof*t_f_wing*rho_mat(1)... % 2 wing flange pieces (on upper and lower side of wing)
        +4*h_prof*t_sp_wing*rho_mat(1) + 4*h_prof*t_spsw_wing*rho_mat(5)... % wing spar and wing spar sandwich. Why 4 spar pieces and not two?
        +s0_LE*c*(t_LE_wing*rho_mat(mat_LE_wing)+t_LEsw_wing*rho_mat(5))... % leading edge
        +s0_TE*c*(t_TE_wing*rho_mat(mat_TE_wing)+t_TEsw_wing*rho_mat(5)))... % trailing edge
        +1.2*(N+1)/(N)*(0.35*c-0.0914*c-h_prof/2)*h_prof*0.7*(t_r1_wing*rho_mat(1)+t_r1sw_wing*rho_mat(5))... %Front part of rib, assumes sandwich structure
        +1.2*(N+1)/(N)*(0.65*c-(1-0.8675)*c-h_prof/2)*h_prof*0.4*(t_r2_wing*rho_mat(1)+t_r2sw_wing*rho_mat(5))); %Rear part of rib, assumes sandwich structure
```

You can get some (but only a superficial) understanding on how this is implemented from page 17 of https://www.research-collection.ethz.ch/bitstream/handle/20.500.11850/90524/eth-46751-02.pdf . The thing is that the first 5 elements that are summed (wing skin ... trailing edge) need to be multiplied by _delta_ (the distance between the discretization points along the wing span), whereas the rib weight can just be added to each discretizatino point without multiplying by delta.

Also, it is correct that you removed the additional paranthesis after `rho_mat(mat_TE_wing)` in line 849, however this needs to be inserted after `t_TEsw_wing*rho_mat(5)` in the same line. 

Do you want to adapt all this or shall I?

Reply:
I will not adapt this part since I don't fully understand it :)

Reply:
TODO (note for myself): Adapt this in a later PR.